Zoom Buttons/ZoomButtons.js

```diff
@@ -0,0 +1,201 @@
+            }
+
+            // initially we have to open the menu and wait until the slider is loaded in dom, or reading slider does not work
+            // try x times and then give up, maybe its dashboard and we dont want to poll
+
+            // initialize when buttons are rendered, which is also boardchange
+            waitForSlider(10); // try 10 times and then give up, might be dashboard
+
+        };
+
+        var toolbarComponents = [
+            'board.toolbar'
+        ];
+
+        toolbarComponents.forEach(function(componentName) {
+            addBusListener(componentName, 'afterRender', appendZoomButtons);
+        });
+
+        $(document).on('click','.btnZoom', function(){
+            var zoom = $(this).attr('data-zoom');
+
+            var was_open = false;
+            if ( isActionMenuOpen() ) {
+                was_open = true;
+            }
+
+            if ( !was_open ) openActionMenu();
+
+            // console.log( "was open btnZoom: ", was_open );
+
+            $('.ui-slider--zoomer').slider('value', zoom);                     // set zoom
+
+            // update highlight
+            setZoomHighlight( zoom );
+
+            if ( !was_open ) closeActionMenu();
+
+        });
+
+        var checkZoomButton = function(){
+
+            var was_open = false;
+            if ( isActionMenuOpen() ) {
+                was_open = true;
+            }
+
+            // console.log( "checked: ", was_open );
+
+            if ( !was_open ) openActionMenu();
+
+            if ( $('.ui-slider--zoomer').length ) {
+                var zoom = $('.ui-slider--zoomer').slider('value');                     // read zoom from slider
+                setZoomHighlight( zoom );
+            }
+            if ( !was_open ) closeActionMenu();
+
+            // try again when menu is open
+            if ( was_open ) {
+                zoomTimer = setTimeout(checkZoomButton, 1000);  // try again, slider was not ready
+            }
+        };
+
+        var waitForSlider = function( tries ) {
+
+            openActionMenu();
+
+            // console.log( 'wait before');
+
+            if ( $('button.tau-btn.tau-btn--view-switch.i-role-board-tooltip.tau-btn-list-view.tau-checked').length ) {
+                // list view, just close and skip
+                // hide buttons ?
+                $('#zoom-buttons').hide();
+                closeActionMenu();
+                return false;
+            }
+
+
+
+            if ( $('.ui-slider--zoomer').length ) {
+                // console.log( 'wait inner');
+
+                var zoom = $('.ui-slider--zoomer').slider('value');                     // read zoom from slider
+                setZoomHighlight( zoom );
+
+
+                closeActionMenu();
+
+                // start polling sync when menu is toggled, only open, only if triggered manually
+                $('.tau-board-header__control--actions .tau-actions-btn:not(.tau-checked)').on('click', function() {
```

Comment:
Please, extract `.tau-board-header__control--actions .tau-actions-btn` to a constant and reuse everywhere.

In case we change that class name in future, that will require to fix one line instead of many ones.